packages/js/email-editor/package.json

```diff
@@ -0,0 +1,91 @@
+		"lint:js": "wp-scripts lint-js './src/**/*.{js,jsx,json,ts,tsx,yml,yaml}'",
+		"lint:js-fix": "wp-scripts lint-js './src/**/*.{js,jsx,json,ts,tsx}' --fix",
+		"check-types": "NODE_OPTIONS=--max_old_space_size=2048 tsc --noEmit",
+		"format": "./node_modules/.bin/prettier --config .prettierrc.js --write 'src/**/*.{js,jsx,json,ts,tsx,yml,yaml,scss}'",
+		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",
+		"build:project": "pnpm --if-present '/^build:project:.*$/'",
+		"build:project:editor": "pnpm build:editor",
+		"lint": "pnpm --if-present '/^lint:lang:.*$/'",
+		"lint:fix": "pnpm --if-present '/^lint:fix:lang:.*$/'",
+		"lint:fix:lang:js": "pnpm lint:js-fix",
+		"lint:lang:js": "pnpm lint:js",
+		"prepack": "pnpm build",
+		"watch:build": "pnpm --if-present --workspace-concurrency=Infinity --filter=\"$npm_package_name...\" --parallel '/^watch:build:project:.*$/'",
+		"watch:build:project": "pnpm --if-present run '/^watch:build:project:.*$/'",
+		"watch:build:project:editor": "pnpm start"
+	},
+	"lint-staged": {
+		"*.{js,jsx,json,ts,tsx,yml,yaml,scss}": "pnpm run format",
+		"*.{scss,css}": "pnpm run lint:css",
+		"*.{js,jsx,ts,tsx}": "pnpm run lint:js"
+	},
+	"dependencies": {
+		"@wordpress/api-fetch": "wp-6.6",
+		"@wordpress/base-styles": "wp-6.6",
+		"@wordpress/block-editor": "wp-6.6",
+		"@wordpress/block-library": "wp-6.6",
+		"@wordpress/blocks": "wp-6.6",
+		"@wordpress/components": "wp-6.6",
+		"@wordpress/compose": "wp-6.6",
+		"@wordpress/core-data": "wp-6.6",
+		"@wordpress/data": "wp-6.6",
+		"@wordpress/data-controls": "wp-6.6",
+		"@wordpress/edit-post": "wp-6.6",
+		"@wordpress/editor": "wp-6.6",
+		"@wordpress/element": "wp-6.6",
+		"@wordpress/format-library": "wp-6.6",
+		"@wordpress/hooks": "wp-6.6",
+		"@wordpress/html-entities": "wp-6.6",
+		"@wordpress/i18n": "wp-6.6",
+		"@wordpress/icons": "wp-6.6",
+		"@wordpress/interface": "wp-6.6",
+		"@wordpress/is-shallow-equal": "wp-6.6",
+		"@wordpress/keyboard-shortcuts": "wp-6.6",
+		"@wordpress/keycodes": "wp-6.6",
+		"@wordpress/media-utils": "wp-6.6",
+		"@wordpress/notices": "wp-6.6",
+		"@wordpress/preferences": "wp-6.6",
+		"@wordpress/private-apis": "wp-6.6",
+		"@wordpress/rich-text": "wp-6.6",
+		"@wordpress/url": "wp-6.6",
+		"classnames": "^2.3.2",
+		"deepmerge": "^4.3.1",
+		"lodash": "^4.17.21",
+		"react": "18.3.x",
+		"react-dom": "18.3.x"
+	},
+	"devDependencies": {
+		"@types/history": "^4.7.11",
+		"@types/js-cookie": "^3.0.2",
+		"@types/lodash": "^4.14.180",
+		"@types/node": "20.x.x",
+		"@types/react": "18.3.x",
+		"@types/react-dom": "18.3.x",
+		"@types/wordpress__block-editor": "^11.5.14",
+		"@types/wordpress__block-library": "^2.6.3",
+		"@types/wordpress__blocks": "^12.5.14",
+		"@types/wordpress__edit-post": "^7.5.7",
+		"@types/wordpress__editor": "^13.6.8",
+		"@types/wordpress__media-utils": "^4.14.4",
+		"@woocommerce/eslint-plugin": "^2.3.0",
```

Comment:
All `@woocommerce/*` deps should be using version `workspace:*` for proper integration with monorepo.